qiskit/visualization/pulse_visualization.py

```diff
@@ -42,7 +42,7 @@ def pulse_drawer(data, dt=1, style=None, filename=None,
         filename (str): Name required to save pulse image
         interp_method (Callable): interpolation function
             See `qiskit.visualization.interpolation` for more information
-        scale (float): scaling of waveform amplitude
+        scale (float or dict): scaling of waveform amplitude
```

Comment:
a scale as a float is self-explanatory. A scale as a dict might not be. Expand the explanation for that case.